myPortfolio.html

```diff
@@ -6,32 +6,35 @@
 	<link href="normalise.css" type="text/css" rel="stylesheet">
 	<link href="myPortfolio.css" type="text/css" rel="stylesheet">
 	<link href="https://fonts.googleapis.com/css2?family=Kufam:wght@400;600&display=swap" rel="stylesheet">
-	<meta name=’viewport” content=”width=device-width, initial-scale=0.1”>
+	<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+	      rel="stylesheet">
+	<meta name=’viewport” content=”width=device-width, initial-scale=1.0”>
+	
 </head>
 <body>
 	<nav class="navigation">
-		<a href="#">About Me</a>
-		<a href="#">Portfolio</a>
-		<a href="#">Contact</a>
+		<a href="#jumpToAboutMe">About Me</a>
+		<a href="#jumpToPortfolio">Portfolio</a>
+		<a href="#jumpToContact">Contact</a>
```

Comment:
You might like to consider a media query for the height of the nav when the screen is narrow
![Screenshot 2020-09-09 at 15 11 43](https://user-images.githubusercontent.com/65677163/92610109-3e800f80-f2af-11ea-92be-ccac7768d2f2.png)